javascript-data-model/order-history.js

```diff
@@ -0,0 +1,95 @@
+var orderHistory = [
+  {
+    orderPlacementDate: [8, 4, 2020],
+    orderDeliveryDate: [8, 8, 2020],
+    totalPrice: ['$', 34.00],
+    shippedTo: 'JS Masher',
+    orderNumber: [114, 3941689, 8772232],
+    handedDirectly: false,
+    // numberOfItems: 1,
```

Comment:
If you wonder why I commented out this property, is because I realised that `orderHistory[0].item.length` would serve the same purpose (same reason I did not specify a property for number of authors)